### ISSUE TITLE
Bad argument

### ISSUE THREAD

**Issue by jasonkuhrt** [AtomLinter/linter-flow#30]
Seems like #8 but that is very old. I'm on release `v3.1.0`. 

```
TypeError: Bad argument
  at TypeError (native)
  at ChildProcess.spawn (internal/child_process.js:274:26)
  at exports.spawn (child_process.js:343:9)
  at /Users/jasonkuhrt/.atom/packages/linter-flow/build/index.js:90:50
  at Object.lint (/Users/jasonkuhrt/.atom/packages/linter-flow/build/index.js:89:16)
  at Promise.then._this.emitter.emit.linter (/Users/jasonkuhrt/.atom/packages/linter/lib/linter-registry.coffee:59:22)
  at LinterRegistry.triggerLinter (/Users/jasonkuhrt/.atom/packages/linter/lib/linter-registry.coffee:58:16)
  at /Users/jasonkuhrt/.atom/packages/linter/lib/linter-registry.coffee:52:17
  at Array.map (native)
  at /Users/jasonkuhrt/.atom/packages/linter/lib/linter-registry.coffee:50:27
(anonymous function) @ /Users/jasonkuhrt/.atom/packages/linter-flow/build/index.js:121
```

**Comment by jonS90** [comment]
Exact same stack trace here. Just installed linter-flow for the first time. (Same package version as above.)

**Comment by izaakschroeder** [comment]
:+1:

**Comment by steelbrain** [comment]
Is your config `linter-flow.pathToFlowExecutable` correct?

**Comment by nmn** [comment]
In order to fix this, can you:
1) make sure flow is available on path to linter-flow. You can use `which flow` to get an absolute path and paste that in the config for linter-flow.
2) Make sure the `.flowconfig` exists in your project and flow works from the cli.

If that that still doesn't work, please let me know.

**Comment by nikgraf** [comment]
I have the same issue:

```
TypeError: Bad argument
  at TypeError (native)
  at ChildProcess.spawn (internal/child_process.js:274:26)
  at exports.spawn (child_process.js:343:9)
  at /Users/nik/.atom/packages/linter-flow/build/index.js:90:50
  at Object.lint (/Users/nik/.atom/packages/linter-flow/build/index.js:89:16)
  at Promise.then._this.emitter.emit.linter (/Users/nik/.atom/packages/linter/lib/linter-registry.coffee:59:22)
  at LinterRegistry.triggerLinter (/Users/nik/.atom/packages/linter/lib/linter-registry.coffee:58:16)
  at /Users/nik/.atom/packages/linter/lib/linter-registry.coffee:52:17
  at Array.map (native)
  at /Users/nik/.atom/packages/linter/lib/linter-registry.coffee:50:27
(anonymous function) @ /Users/nik/.atom/packages/linter-flow/build/index.js:121
```

Run the flow command from the root:

```
➜  time git:(master) flow
Flow server launched for /Users/nikgraf/Projects/time
Spawned flow server (child pid=88063)
Logs will go to /private/tmp/flow/zSUserszSnikgrafzSProjectszStime.log
The Flow server will be ready in a moment.
No errors!r still initializing -- this can take some time. Retrying... \
```

and used the absolute path & rebooted atom:

```
➜  time git:(master) which flow
/usr/local/bin/flow
```

<img width="771" alt="screen shot 2015-09-23 at 21 24 22" src="https://cloud.githubusercontent.com/assets/223045/10056057/d8ce3938-6239-11e5-8b06-eb073335aff7.png">

**Comment by nmn** [comment]
looking into it. You seem to be set up correctly.

**Comment by nmn** [comment]
found the issue, will push a fix in an hour. I hope I'm right about the problem.

**Comment by nmn** [comment]
Just published a patch. Please confirm that the problem has been fixed.

**Comment by gigq** [comment]
I'm still getting this issue in 3.1.2

**Comment by sambev** [comment]
@gigq I was getting this too.
I could be totally off here, but when I followed the trace, I noticed that a variable called `cmdString` was getting passed to `child_process.spawn` and was undefined. Looking at the assignment of `cmdString` I found this:

`    cmdString = atom.config.get('linter-flow-plus.pathToFlowExecutable');`

But I was not running linter-flow-plus, I was running linter-flow. I was pretty confused so I removed linter-flow and re installed and it worked. I really have no idea what was going on there
